community/modules/compute/schedmd-slurm-gcp-v6-partition/outputs.tf

```diff
@@ -18,18 +18,16 @@ output "partitions" {
   value = [local.partition]
 
   precondition {
-    condition     = (length(local.use_placement) == 0) || var.exclusive
-    error_message = "If any nodeset `enable_placement`, `var.exclusive` must be set true"
+    condition     = (length(local.non_static_ns_with_placement) == 0) || var.exclusive
+    error_message = "If any non-static nodesets has `enable_placement`, `var.exclusive` must be set true"
```

Comment:
nit: I would use the terminology "autoscaling only" in place of "non-static", but just personal preference.

Reply:
ack